### ISSUE TITLE
Color Contrast Issue – Empty Hint Text

### ISSUE THREAD

**Issue by X-07-xqf** [X-07-xqf/2850final-project#75]
The empty meal hint text does not meet WCAG 2.1 AA contrast requirements.
